lib/puppet/functions/fqdn_rotate.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# @summary DEPRECATED.  Use the namespaced function [`stdlib::fqdn_rotate`](#stdlibfqdn_rotate) instead.
+Puppet::Functions.create_function(:fqdn_rotate) do
+  dispatch :deprecation_gen do
+    repeated_param 'Any', :args
+  end
+  def deprecation_gen(*args)
+    call_function('deprecation', 'fqdn_rotate', 'This method is deprecated, please use stdlib::fqdn_rotate instead.')
```

Comment:
With your work on #1377, I think it makes sense to:

```suggestion
    call_function('deprecation', 'fqdn_rotate', 'This method is deprecated, please use stdlib::fqdn_rotate instead.', false)
```

We may also open another PR to be merged when the next major version is released and that restore the failure?

Reply:
Fixed whilst rebasing by rerunning script.